src/main/kotlin/com/demonwav/mcdev/platform/mcp/inspections/StackEmptyInspection.kt

```diff
@@ -0,0 +1,100 @@
+/*
+ * Minecraft Dev for IntelliJ
+ *
+ * https://minecraftdev.org
+ *
+ * Copyright (c) 2018 minecraft-dev
+ *
+ * MIT License
+ */
+
+package com.demonwav.mcdev.platform.mcp.inspections
+
+import com.demonwav.mcdev.util.fullQualifiedName
+import com.intellij.codeInspection.ProblemDescriptor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.*
+import com.siyeh.ig.BaseInspection
+import com.siyeh.ig.BaseInspectionVisitor
+import com.siyeh.ig.InspectionGadgetsFix
+import org.jetbrains.annotations.Nls
+
+class StackEmptyInspection : BaseInspection() {
+    companion object {
+        const val STACK_FQ_NAME = "net.minecraft.item.ItemStack"
+        const val EMPTY_NAME = "EMPTY"
```

Comment:
It's highly unlikely that this MCP name is going to change, but we might want to cover it using SRG names (which are also subject to change, but even less likely). See the following snippet for an example:
https://github.com/minecraft-dev/MinecraftDev/blob/dev/src/main/kotlin/com/demonwav/mcdev/i18n/translations/TranslationFunction.kt#L38-L42
I'm not sure if that's the best example, but the only one in the codebase I know off the top of my head.

Reply:
Classes do have SRG names, the SRG names just happen to match MCP names. In this case using the SRG name for the field would be more robust.